docs/README_DE.md

```diff
@@ -27,29 +27,53 @@
-- Simulation der Rückstrahlzone
-- Ein Feuerleitsystem für gepanzerte Fahrzeuge und Hubschrauber  
-***und noch viel mehr...***
+- Sprengtoffmechaniken mit den unterschiedlichsten Zündern
+- Kartenverbesserungen – Setzen von Markierungen ; Kartenwerkzeuge
+- Erweitertes Raketenlenksystem 
+
+#### Weitere Mechaniken
+- Tragen und Ziehen
+- Waffen und Fahrzeuge tragen die Namen ihrer Vorbilder in der echten Welt
+- Ein Feuerleitsystem (FLS) für Hubschrauber und Panzer
+- Viele Funktionen werden in C/C++ Erweiterungen berechnet
+- Rückstrahlzonen- und Überdrucksimulation
+- Einwegwaffen
+- Realistische G-Kräfte
+- Fahrzeuge abschließen
+- Realistische Nacht- und Thermalsicht
+- Magazin umpacken
+- Realistische Waffen Er- bzw. Überhitzung
+- Temporäre Taubheit bei zu lauten Geräuschen 
+- Verbesserte Ragdollphysik
+- Verbesserte Interaktionen für AARs und Munitionsschlepper
+- Einstellbare Zielfernrohre
+- Keine Ruheanimationen bei gesenkter Waffe
+- Über Hindernisse springen, über Mauern klettern, Zäune durchtrennen
+- Keine "sprechender Charkater"
+- Vector IV, MicroDAGR und Kestrel <br>
+***und noch viel viel mehr...***
 
 #### Anleitungen
 Du hast ACE3 installiert, hast aber keine Ahnung was und wie alles funktioniert und wo sich was befindet?
 - [Erste Schritte](http://ace3mod.com/wiki/user/getting-started.html).
 
 #### Mitwirken
-Wenn du bei der Entwicklung der MOD mithelfen möchtest, so kannst du dies tun, indem du nach Fehlern Ausschau hältst, oder neue Funktionen vorschlägst. Um etwas beizutragen, "Forke" einfach dieses Archiv (bzw. repository) und erstelle deine "Pull-Request", welche von anderen Entwicklern und Beiträgern überprüft wird. Bitte trage dich dabei in [`AUTHORS.txt`](https://github.com/acemod/ACE3/blob/master/AUTHORS.txt) mit deinem Nutzernamen und einer gütligen Email-Adresse ein.  
+Wenn du bei der Entwicklung der MOD mithelfen möchtest kannst du dies tun, indem du nach Fehlern Ausschau hältst oder neue Funktionen vorschlägst. Um etwas beizutragen, "Forke" einfach dieses Archiv (bzw. "repository") und erstelle deine "Pull-Request", welche von anderen Entwicklern und Beiträgern überprüft wird. Bitte trage dich dabei in [`AUTHORS.txt`](https://github.com/acemod/ACE3/blob/master/AUTHORS.txt) mit deinem Nutzernamen und einer gütligen Email-Adresse ein.  
 
-Um einen Fehler oder ein Feature zu melden bzw. ein bereits Bestehendes zu ändern - nutze unseren [Issue Tracker](https://github.com/acemod/ACE3/issues). Besuche auch:
+Um einen Fehler, Anregungen oder neue Funktionalitäten uns mitzuteilen: Nutze unseren [Issue Tracker](https://github.com/acemod/ACE3/issues). Besuche auch:
 - [Wie kann ich ein Problem melden](http://ace3mod.com/wiki/user/how-to-report-an-issue.html)
-- [Wie kann ich ein Feature Request erstellen](http://ace3mod.com/wiki/user/how-to-make-a-feature-request.html)
+- [Wie kann ich ein Wunsch zu einer neuen Funktion mitteilen? ](http://ace3mod.com/wiki/user/how-to-make-a-feature-request.html)
```

Comment:
Bei der Bearbeitung der Datei konnte ich kein Lehrzeichen feststellen. Dementsprechend: Ready to merge. Außerdem handelt es sich hierbei um keinen Quellcode. Deshalb halte ich ein eventuelles Leerzeichen für vernachlässigbar. 
Wie schon im anderen PL gesagt. Alles andere wird am Wocheende berarbeitet.